jlm/mlir/backend/JlmToMlirConverter.cpp

```diff
@@ -13,6 +16,9 @@
 #include <llvm/Support/raw_os_ostream.h>
 #include <mlir/IR/Verifier.h>
 
+#include "JlmToMlirConverter.hpp"
```

Comment:
Please use `#include <...>`

Reply:
The same header is already included at line 7, no? This line can be removed!?

Reply:
Has been removed